spec/query_spec.cr

```diff
@@ -588,4 +588,27 @@ describe Avram::Query do
       UserQuery.new.select_count.should eq 0
     end
   end
+
+  describe "delete_all" do
+    it "uses the glove" do
```

Comment:
I’m sure this is a reference I’m not cool enough to understand 😆 could you also add a real description before or after?

Reply:
😱